Document/0x06c-Reverse-Engineering-and-Tampering.md

```diff
@@ -316,4 +316,93 @@ Cycript tricks:
 
 http://iphonedevwiki.net/index.php/Cycript_Tricks
 
-<!-- TODO [Develop section on Frida] -->
+#### Frida
+
+It’s Greasemonkey for native apps, or, put in more technical terms, it’s a dynamic code instrumentation toolkit. It lets you inject snippets of JavaScript or your own library into native apps on Windows, macOS, Linux, iOS, Android, and QNX. Frida also provides you with some simple tools built on top of the Frida API. These can be used as-is, tweaked to your needs, or serve as examples of how to use the API.
+
+##### Installation
+
+*Requirements for Python bindings*
+
+Installing Frida is easy and straight-forward, but there are a few requirements you’ll need to make sure your system has before you start.
```

Comment:
Please just use a sentence like the following instead of copying it into the MSTG:
The documentation of Frida does describe on [how to install and verify the installation](https://www.frida.re/docs/installation/ "Installation of Frida"). 

If the original documentation is good, link to it and don't copy the content. Then we also don't need to maintain updates in the MSTG. Thanks